src/components/card/card.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+function Card(): React.ReactElement {
```

Comment:
```suggestion
export const Card: FC = () => {
```
При составлении функционального компонента ты не будет использовать слово this, то правильнее это показать через экспорт стрелочной функции.